src/modules/fuzzy-search/fuzzy-search.controller.ts

```diff
@@ -0,0 +1,33 @@
+import { Body, Controller, Get, Param, Put } from '@nestjs/common';
+import { FuzzySearchUseCases } from './fuzzy-search.usecase';
+import { ApiProperty, ApiTags } from '@nestjs/swagger';
+import { User } from '../user/user.domain';
+import { User as UserDecorator } from '../auth/decorators/user.decorator';
+
+class RequestBody {
+  @ApiProperty() fileId: string;
+  @ApiProperty() name: string;
+}
+
+@ApiTags('Fuzzy')
+@Controller('fuzzy')
+export class FuzzySearchController {
+  constructor(private readonly usecases: FuzzySearchUseCases) {}
+
+  @Get('/:search')
+  async fuzzySearch(
+    @UserDecorator() user: User,
+    @Param('search') search: string,
+  ) {
+    return await this.usecases.fuzzySearch(user.uuid, search);
+  }
+
+  @Put('/')
+  async add(@UserDecorator() user: User, @Body() content: RequestBody) {
```

Comment:
We are going to add the item to the look up table once the resource is created (file/folder creation)